src/DynamoPackages/PackageLoader.cs

```diff
@@ -134,7 +134,14 @@ public void Load(Package package)
                 {
                     if (assem.IsNodeLibrary)
                     {
-                        OnRequestLoadNodeLibrary(assem.Assembly);
```

Comment:
I wonder why, if there is no ZT library, does the `assembly.IsNodeLibrary` condition turn out to be true?